common/persistence/nosql/nosqlplugin/cassandra/gocql/client.go

```diff
@@ -150,6 +150,12 @@ func NewCassandraCluster(
 	cluster.Consistency = cfg.Consistency.GetConsistency()
 	cluster.SerialConsistency = cfg.Consistency.GetSerialConsistency()
 
+	cluster.ReconnectionPolicy = &gocql.ExponentialReconnectionPolicy{
+		MaxRetries:      30,
+		InitialInterval: time.Second,
+		MaxInterval:     time.Minute,
```

Comment:
max interval should not be as high as 1min, let us try 10s?

Reply:
good point. fixed